src/App.tsx

```diff
@@ -1,10 +1,13 @@
-import { useState } from 'react'
+//import { useState } from 'react'
 import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import './App.css'
+import {useCountStore} from './store/store';
```

Comment:
Alright, I tried src/store cuz we included src in the config.. but it didn't work

Reply:
it didn't work because you are inside `src`, so if you wrote `src/store` this will mean from the root project point of view `src/src/store` that's why it did not work.

I'm fine with relative file paths, my point is about `store/store` there is no need for it.
We can eliminate it by using `index.ts`.

Reply:
Done